backend/pkg/sessions/sessions.go

```diff
@@ -254,6 +256,18 @@ func (s *sessionsImpl) UpdateEventsStats(sessionID uint64, events, pages int) er
 	return nil
 }
 
+func (s *sessionsImpl) IsExist(projectID int, sessionID string) bool {
+	sessionIDUint, err := strconv.ParseUint(sessionID, 10, 64)
```

Comment:
Why you don't want to pass the sessionID as an uint64 as we have it in all other methods?